tests/source/issue-1124.rs

```diff
@@ -0,0 +1,3 @@
+use d; use c; use b; use a; 
```

Comment:
The current test has a space at the end of the first line already - I'll add a comment (*after* the `use` line - the whole point was putting that line at the start of the file!) to highlight that.

As for a test with blank lines - yes, no problem. That will reflect the current behaviour of the import reordering, which doesn't eliminate blank lines (as @crumblingstatue suggests, and which I think is worth looking at), but keeps them in verbatim.